x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/high_count_by_user_agent_jsbase.json

```diff
@@ -0,0 +1,38 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "APM JSBase: Detects user agents that are making requests at a suspiciously high rate. This is useful in identifying bots.",
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "high_non_zero_count partitionfield=\"user_agent.name\"",
```

Comment:
`"description": "high request rate for user agent"`

Reply:
https://github.com/elastic/kibana/pull/53792/commits/ebe6353130f170600b92e45be911f70264e4f4be